src/equality_metrics.jl

```diff
@@ -52,6 +52,9 @@ end
 _psnr(ref::AbstractArray{<:Color3}, x::AbstractArray{<:Color3}) =
     _psnr(channelview(RGB.(ref)), channelview(RGB.(x)), 1.0)
 
+_psnr(ref::AbstractArray{<:ColorTypes.RGBA}, x::AbstractArray{<:ColorTypes.RGBA}) =
+    _psnr(channelview(ref), channelview(x), 1.0)
```

Comment:
Maybe

```suggestion
_psnr(ref::AbstractArray{<:ColorTypes.Transparent3}, x::AbstractArray{<:ColorTypes.Transparent3}) =
    _psnr(channelview(ARGB.(ref)), channelview(ARGB.(x)), 1.0)
```

Reply:
I'm not familiar enough with `ColorTypes` to know which would be the better way.

Reply:
There're also other transparent types, e.g., both `RGBA` and `ARGB` exist. I'm not an expert either but I think this is more general.